packages/govern-core/test/pipelines/govern-queue.test.ts

```diff
@@ -34,6 +35,25 @@ describe('Govern Queue', function () {
     const GQ = (await ethers.getContractFactory(
       'GovernQueue'
     )) as GovernQueueFactory
-    gq = await GQ.deploy(ownerAddr, INIT_CONFIG)
+    gq = await GQ.deploy(ownerAddr, INIT_CONFIG) as GovernQueue
+  })
+
+  context('ERC-165', () => {
+    const ERC165_INTERFACE_ID = '0x01ffc9a7'
+    const ERC3000_INTERFACE_ID = '0x6d74ef88'
+
+    // TODO: Fix types @gabi
```

Comment:
I review and ran the tests and the types are fine to me. What you mean?

Reply:
I fixed it by adding the `expect` import that was missing and ts was giving me some weird issues

Reply:
Oh right, with the addition of typescript buidler required to be explicit about the imports.